### PR TITLE
Add New Online Competition : Paldea Prologue

### PR DESCRIPTION
**Pull request by rascypoke**
I would like to add a new online competition: "Paldea Prologue" to our format. The official tournament outline is as follows.
https://sv-news.pokemon.co.jp/en/page/39.html

This is the first revision I am implementing, so please point out any bad parts.

### CODE REVIEW

**Review by sabertazimi — CHANGES_REQUESTED** [review]
https://github.com/smogon/pokemon-showdown/blob/79693ab2dd2ecb651ddeaf81ef4ebea383dc5438/data/rulesets.ts#L31-L37

https://github.com/smogon/pokemon-showdown/blob/79693ab2dd2ecb651ddeaf81ef4ebea383dc5438/data/rulesets.ts#L53-L67

**Comment by rascypoke** [comment]
Fixed, thanks for pointing it out.

**Review by sabertazimi — CHANGES_REQUESTED** [review]
```bash
CRASH: Error: Rule "!!adjustlevel=50" is not replacing anything (it should not have "!!")
    at DexFormats.getRuleTable (/home/sabertaz/git/pokemon-showdown/dist/sim/dex-formats.js:578:21)
    at get formatListText [as formatListText] (/home/sabertaz/git/pokemon-showdown/dist/server/rooms.js:1222:37)
    at GlobalRoomState.handleConnect (/home/sabertaz/git/pokemon-showdown/dist/server/rooms.js:1459:82)
    at Object.socketConnect (/home/sabertaz/git/pokemon-showdown/dist/server/users.js:1303:16)
    at Object.onSpawn (/home/sabertaz/git/pokemon-showdown/dist/server/sockets.js:63:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by Karthik99999** [comment]
Take a look at the VGC 2022 format to implement this. You should be using `Flat Rules` and putting `Restricted Legendary` in the `restricted` array. Also please use tabs for indentation.

**Comment by rascypoke** [comment]
Sorry for the repetition, I have committed again.

File: config/formats.ts [hunk 1]
```diff
@@ -259,6 +259,14 @@ export const Formats: FormatList = [
 		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer', 'Open Team Sheets'],
 		banlist: ['Sub-Legendary'],
 	},
+	{
+		name: "[Gen 9] Paldea Prologue",
+
+		mod: 'gen9',
+		gameType: 'doubles',
+		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer'],
+		restricted: ['Limit One Restricted'],
```
Comment (Karthik99999) [inline config/formats.ts:268]:
```suggestion
		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer', 'Limit One Restricted'],
		restricted: ['Restricted Legendary'],
```

**Review by sabertazimi — CHANGES_REQUESTED** [review]
```js
	{
		name: "[Gen 9] Paldea Prologue",

		mod: 'gen9',
		gameType: 'doubles',
		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer', 'Limit One Restricted'],
		restricted: ['Restricted Legendary'],
	},

```

**Pull request merged**